src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -231,16 +231,16 @@ public abstract class TreeOptions : LearnerInputBaseWithGroupId
         public int MaxCategoricalSplitPoints = 64;
 
         /// <summary>
-        /// Minimum categorical docs percentage in a bin to consider for a split.
+        /// Minimum categorical example percentage in a bin to consider for a split.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "Minimum categorical docs percentage in a bin to consider for a split.", ShortName = "mdop")]
-        public double MinDocsPercentageForCategoricalSplit = 0.001;
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "Minimum categorical docs percentage in a bin to consider for a split.", ShortName = "mdop,MinDocsPercentageForCategoricalSplit")]
+        public double MinExamplePercentageForCategoricalSplit = 0.001;
```

Comment:
>0.001 [](start = 64, length = 5)

is it 0.001% or 0.1% ?

Reply:
0.1%.

---
In reply to: [260563129](https://github.com/dotnet/machinelearning/pull/2753#discussion_r260563129) [](ancestors = 260563129)